repos/spack_repo/builtin/packages/libemos/package.py

```diff
@@ -25,12 +25,7 @@ class Libemos(CMakePackage):
     version("4.4.7", sha256="669fb070c1ce655812882140a92b100233f065829868d9374bad2fcbb6b356e5")
     version("4.4.2", sha256="e2d20ad71e3beb398916f98a35a3c56ee0141d5bc9b3adff15095ff3b6dccea8")
 
-    variant(
-        "grib",
-        default="eccodes",
-        values=("eccodes", "grib-api"),
-        description="Specify GRIB backend",
-    )
+    variant("grib", default="eccodes", values=("eccodes",), description="Specify GRIB backend")
```

Comment:
This variant should have a version guard so that we don't propagate the variant with only one value to newer versions of libemos.

```suggestion
    variant("grib", default="eccodes", values=("eccodes",), when="@:4.5.1", description="Specify GRIB backend")
```

Reply:
We don't have a formal variant deprecation procedure. I would rather just delete the variant.